catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -114,6 +116,10 @@
 
   private MetaTypeService metaTypeService;
 
+  // age off entries after 3 seconds which should be enough time for the remote systems solr to
+  // persist the entry
+  private Map<String, Metacard> recent = new PassiveExpiringMap<>(TimeUnit.SECONDS.toMillis(3));
```

Comment:
❓ Does this account for remote systems with low-bandwidth?

Reply:
Since the create request is within the sync block the first request should have completed before another one could be attempted so the bandwidth should not factor in. If anything low bandwidth connections should have less of a problem because that gives the remote system more time to persist the entry. Under "normal" conditions solr can take up to 1 sec to persist so I went with 3x that to be on the safe side.